vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -536,12 +535,14 @@ export default {
             : "NA"
         );
 
+        formData.append("resume", this.file, fileName);
+        formData.append("position", this.job.title);
```

Comment:
Any need for this as we are setting it as `job_title`?

Reply:
no need extra, I thought that when we remove the email sending functionality, the other member will easily understand that we have used this job title value for the position.